webapp/src/components/expanded_view/component.tsx

```diff
@@ -684,6 +684,7 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
                                     />
                                 }
                                 unavailable={noScreenPermissions}
+                                disabled={sharingID !== '' && !isSharing}
```

Comment:
We were actually forgetting to disable it here.

Reply:
I tested, and it looks like it hides it only when you haven't given permissions (not when you press cancel), so that could be a way to distinguish between the two error states? (not 100% sure, only a quick test)

Reply:
Mmm, not sure I understand. What this does is it disables the button when someone else is screen sharing.

Reply:
Hahah... ahh -- I misunderstood. It thought it hides when you don't have permissions to share, because I used Opera to test it and I was logged into a server with one team (smaller LHS), which hides the screensharing button. 🤕  I'm going to fade back into the bushes now.

Reply:
Opera? Good Lord :see_no_evil: